ambry-network/src/main/java/com.github.ambry.network/ResponseInfo.java

```diff
@@ -18,7 +18,7 @@
 
 
 /**
- * The response from a {@link NetworkClient} comes in the form of an object of this class.
+ * The response from a {@link SocketNetworkClient} comes in the form of an object of this class.
```

Comment:
This applies to AzureNetworkClient as well, right?

Reply:
reverted the change